opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -104,10 +145,15 @@ def __init__(
         metric_readers: Sequence[MetricReader] = (),
         resource: Resource = Resource.create({}),
         shutdown_on_exit: bool = True,
+        measurement_consumer_class: MeasurementConsumer = (
+            SerialMeasurementConsumer
+        ),
     ):
         self._lock = Lock()
         self._atexit_handler = None
 
+        self._measurement_consumer = measurement_consumer_class()
```

Comment:
how about have a private method in this file `_default_measurement_consumer() -> MeasurementConsumer` which you can patch in tests.

Reply:
I think we can just patch the class itself in the tests.